test/controllers/AuthenticationSpec.scala

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import akka.stream.Materializer
+import controllers.Assets.Ok
+import controllers.auth.Authorisation
+import org.mockito.ArgumentMatchers.{any, eq => argEq}
+import org.mockito.Mockito._
+import org.scalatest.BeforeAndAfterEach
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.Play
+import play.api.http.Status
+import play.api.libs.json.{JsValue, Json}
+import play.api.mvc.{Action, AnyContent, ControllerComponents, Result}
+import play.api.test.FakeRequest
+import play.api.test.Helpers.stubControllerComponents
+import uk.gov.hmrc.auth.core.authorise.Predicate
+import uk.gov.hmrc.auth.core.retrieve.EmptyRetrieval
+import uk.gov.hmrc.auth.core._
+import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{ExecutionContext, Future}
+
+
+class AuthenticationSpec extends UnitSpec with MockitoSugar with BeforeAndAfterEach with WithFakeApplication {
```

Comment:
Naming should match the name of the trait your testing here  and be 'AuthorisationSpec'